neard/src/cli.rs

```diff
@@ -27,54 +27,96 @@ pub(super) struct NeardCmd {
 impl NeardCmd {
     pub(super) fn parse_and_run() {
         let neard_cmd = Self::parse();
-        neard_cmd.opts.init();
-        info!(target: "neard", "Version: {}, Build: {}, Latest Protocol: {}", crate::NEARD_VERSION, crate::NEARD_BUILD, near_primitives::version::PROTOCOL_VERSION);
-
-        #[cfg(feature = "test_features")]
-        {
-            error!("THIS IS A NODE COMPILED WITH ADVERSARIAL BEHAVIORS. DO NOT USE IN PRODUCTION.");
-
-            if env::var("ADVERSARY_CONSENT").unwrap_or_default() != "1" {
-                error!("To run a node with adversarial behavior enabled give your consent by setting variable:");
-                error!("ADVERSARY_CONSENT=1");
-                std::process::exit(1);
+        let verbose = neard_cmd.opts.verbose.as_deref();
+
+        let rust_log = env::var("RUST_LOG");
+        let rust_log = rust_log.as_ref().map(String::as_str).unwrap_or(near_o11y::DEFAULT_RUST_LOG);
+        let mut env_filter = EnvFilter::new(rust_log);
+        if let Some(module) = verbose {
+            env_filter = env_filter
+                .add_directive("cranelift_codegen=warn".parse().unwrap())
```

Comment:
This one feels like dead code

Reply:
The plan is to avoid making any globally functional changes in this PR. If this is dead code, it should be addressed as a followup.